cclib/parser/gamessukparser.py

```diff
@@ -47,6 +47,20 @@ def before_parsing(self):
     def extract(self, inputfile, line):
         """Extract information from the file object inputfile."""
 
+        # Find the version number and optionally the revision number.
+        if "version" in line:
+            search = re.search(r"\sversion\s*(\d\.\d)", line)
+            if search:
+                self.metadata["package_version"] = search.groups()[0]
+        if "Revision" in line:
+            revision = line.split()[1]
+            # Don't add revision information to the main package
```

Comment:
Why?

Reply:
I don't know what it should be called yet. I can ask Daniel Smith about what provenance information they have at this point, since that's where it will be translated to.